doc/getting_started/using_docker.rst

```diff
@@ -42,26 +42,26 @@ This command runs a new container named ``mytarantool``.
 Docker starts it from an official image named ``tarantool/tarantool:2.6.0``,
 with Tarantool version ``2.6.0`` and all external modules already installed.
 
-Tarantool will be accepting incoming connections on ``localhost:3301``.
-You may start using it as a key-value storage right away.
+Tarantool will accept incoming connections on ``localhost:3301``.
+You can start using it as a key-value storage right away.
 
 Tarantool :ref:`persists data <index-box_persistence>` inside the container.
 To make your test data available after you stop the container,
 this command also mounts the host's directory ``/data/dir/on/host``
 (you need to specify here an absolute path to an existing local directory)
 in the container's directory ``/var/lib/tarantool``
 (by convention, Tarantool in a container uses this directory to persist data).
-So, all changes made in the mounted directory on the container's side
+Through this, all changes made in the mounted directory on the container's side
 are applied to the host's disk.
 
 Tarantool's database module in the container is already
 :doc:`configured </reference/reference_lua/box_cfg>` and started.
-You needn't do it manually, unless you use Tarantool as an
+You don't need to do it manually, unless you use Tarantool as an
 :ref:`application server <app_server>` and run it with an application.
 
 .. NOTE::
 
-    If your container terminates soon after start, follow
+    If your container terminates directly after starting, follow
```

Comment:
AFAIU, the term 'directly' is more about spatial or logical relations, not temporal ones. Is it better to say 'immediatelly' or 'instantly'? (I'm not sure here.)